src/ProjectService.Business/Commands/Project/CreateProjectCommand.cs

```diff
@@ -134,6 +166,19 @@ public OperationResultResponse<ProjectInfo> Execute(ProjectRequest request)
 
             _validator.ValidateAndThrowCustom(request);
 
+            var existUsers = CheckUserExistence(request.Users.Select(u => u.UserId).ToList(), response.Errors);
```

Comment:
I guess you have to do it in validator

Reply:
Also you have to check that all of requested users exist, now if one exists, it will be correct, but other can doesn't exist. Count must be equal request.users.count

Reply:
And why there is not return error if one from users not found?

Reply:
Fixed